cosmos/profiles/base.py

```diff
@@ -180,8 +243,9 @@ def get_profile_file_contents(
             }
         }
 
-        if self.disable_event_tracking:
-            profile_contents["config"] = {"send_anonymous_usage_stats": False}
+        config_vars = self.dbt_config_vars.as_dict()
+        if config_vars:
+            profile_contents["config"] = config_vars
```

Comment:
Would suggest having a check with `if self.dbt_config_vars` and then calling the `as_dict()` method and adding it to the the profile contents.

Reply:
fix